### PR TITLE
Update readme with instructions to run app locally

### PR DESCRIPTION
**Pull request by brianrip**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Roots Check in Tool
+
+### Run locally
+
+Clone repo: `git clone git@github.com:damwhit/check_in_tool.git`
```
Comment (erinnachen) [inline README.md]: @brianrip best to clone the https version since not all people are going to be able to push to that repo.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+Bundle dependencies: `bundle`
+
+Establish database: `rake db:setup`
+
```
Comment (erinnachen) [inline README.md:10]: I think this is going end up looking more like `rake db:{create,migrate}` if someone is coming into a clean setup.

**Comment by NickyBobby** [comment]
I think @erinnachen makes a good point. Also do we want to add a part about how to run the test suite? Or we could add that eventually as our test suite gets more robust. @brianrip

**Comment by brianrip** [comment]
I will change to https,  rake db:{create,migrate}, and add instructions for test suite.

**Pull request merged**